DeviceManager/Logger.py

```diff
@@ -27,7 +31,30 @@ def __init__(self, LOG_LEVEL = logging.DEBUG,
             self.log.setLevel(LOG_LEVEL)
             self.log.addHandler(self.stream)
             self.log.handler_set = True
+
+    def update_log_level(self, LEVEL):
+        levelToName = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']
```

Comment:
what means 'notset'?

Reply:
Notset is an option available configuration log that disable it. The new PR removes this option.